howto/anbox/landing.md

```diff
@@ -0,0 +1,5 @@
+The guides in this section describe how to work with the Anbox runtime.
```

Comment:
We should add an explanation of what we mean with "Anbox runtime".

Reply:
Added